docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,244 @@
+---
+title: Machine ID Getting Started Guide
+description: Getting started with Teleport Machine ID
+---
+
+# Getting Started
+
+In this getting started guide, you will use Machine ID to create a bot user for
+a machine and use that identity to connect to said machine.
+
+Here's an overview of what you will do:
+
+1. Download and install Teleport (=teleport.version=)
+2. Create bot user
+3. Start Machine ID
+4. Use Machine ID
+
+## Prerequisites
+
+Before using Machine ID, you will need an existing Teleport cluster.
+
+If you have not setup a Teleport cluster before, follow the
```

Comment:
Nit, pedantic grammar, I apologize in advance. For what it's worth, "setup" is a noun and "set up" is a verb.

```suggestion
If you have not set up a Teleport cluster before, follow the
```